lib/clients/rtm/client.js

```diff
@@ -617,6 +628,35 @@ RTMClient.prototype.sendMessage = function sendMessage(text, channelId, optCb) {
   }, optCb);
 };
 
+/**
+ * Helper for updating a sent message via the 'chat.update' API call
+ * @param message {object} message The message object to be updated,
+ * see /lib/clients/web/facets/chat.js for more details
+ * @param {Function=} optCb Optional callback
+ */
+RTMClient.prototype.updateMessage = function updateMessage(message, optCb) {
+  var err;
+  var ret;
+
+  if (this.connected) {
+    ret = this._chat.update(message.ts, message.channel, message.text, message.opts, optCb);
```

Comment:
I'm a little unclear on what you mean by MS, but my goal here was to expose the already existing `update` function of the [`ChatFacet`](https://github.com/slackhq/node-slack-sdk/blob/master/lib/clients/web/facets/chat.js#L116).